main.go

```diff
@@ -166,10 +182,72 @@ func main() {
 	e.GET("/", handleMainPageRequest(dbClient))
 	e.GET("/:year/:month", handleMonthRequest(dbClient))
 
+	e.GET("/uiapi/v1/orgaos/resumo/:orgao", getSummaryOfAgency)
+	e.GET("/uiapi/v1/orgaos/salario/:orgao/:year/:month", getSalaryOfAgencyMonthYear)
+	e.GET("/uiapi/v1/entidades/resumo/:estado", getSummaryOfEntitiesOfState)
+	e.GET("/uiapi/v1/orgaos/totals/:orgao/:year", getTotalsOfAgencyYear)
+
 	s := &http.Server{
 		Addr:         fmt.Sprintf(":%d", conf.Port),
 		ReadTimeout:  5 * time.Minute,
 		WriteTimeout: 5 * time.Minute,
 	}
 	e.Logger.Fatal(e.StartServer(s))
 }
+
+// mockando dados
+type State struct {
+	Name      string
+	ShortName string
+	FlagUrl   string
+	Agency    []Agency
+}
+
+type Agency struct {
+	Name          string
+	ShortName     string
+	AgencySummary AgencySummary
+	Employee      []Employee
+}
+
+type Employee struct {
+	Name   string
+	Wage   float64
+	Perks  float64
+	Others float64
+}
+
+type AgencySummary struct {
+	TotalEmployees int
+	TotalWage      float64
+	TotalPerks     float64
+	MaxWage        float64
+}
+
+type AgencyTotalsYear struct {
+	Year        int
+	MonthTotals []MonthTotals
+}
+
+type MonthTotals struct {
+	Month  int
+	Wage   float64
+	Perks  float64
+	Others float64
+}
+
+var (
```

Comment:
Declarar e retornar todas as variáveis dentro dos métodos onde elas são usadas. Se for o caso, pode repetí-las.